jenkins-pipelines/artifacts-amazon2.jenkinsfile

```diff
@@ -8,7 +8,7 @@ artifactsPipeline(
     backend: 'aws',
     region_name: 'eu-west-1',
     provision_type: 'spot_low_price',
-    scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo',
+    scylla_mgmt_repo: 'http://downloads.scylladb.com/rpm/centos/scylladb-manager-2.3.repo',
```

Comment:
Why do we  need scylla_mgmt_repo in all these artifacts?
We need only mgmt_agent, please remove it from here.
Please add it in the test_yaml.

Reply:
defining scylla_mgmt_repo works just as well, since in the absence of mgmt_agent repo sct uses scylla_mgmt_repo as the agent repo. And it works:
https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/Shlomo/job/4.4-ubuntu18-artifact/
https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/Shlomo/job/4.4-centos7-artifact/

I didn't define scylla_mgmt_agent_repo since the artifact pipeline does not have the ability to define it.

Reply:
If you read what I ask, you won't need to define it there.
It needs to go to the yaml.
And remove all the mgmt_repo because it's misleading.

Reply:
I set in the yaml instead. Note that ubuntu 20 does not have a manager 2.3 release.

Reply:
@ShlomiBalalis please send another PR, in that PR set in unbntu20 yaml use_mgmt: false.

Reply:
If it tries to install agent (I don't think it should) set there use_mgmt: false.